src/components/v5/shared/UserAvatars/UserAvatars.tsx

```diff
@@ -39,19 +41,35 @@ const UserAvatars: FC<UserAvatarsProps> = ({
           />
         </li>
       ))}
-      {!!remainingAvatarsCount && showRemainingAvatars && (
-        <li
-          className={clsx(
-            'flex items-center justify-center border border-base-white rounded-full bg-gray-50 text-5 text-gray-700 -ml-3 z-10',
-            {
-              'w-[1.875rem] h-[1.875rem]': size === 'sm',
-              'w-[2.375rem] h-[2.375rem]': size === 'xms',
-            },
-          )}
-        >
-          {`+ ${remainingAvatarsCount}`}
-        </li>
-      )}
+      {!!(remainingAvatarsCountProp || remainingAvatarsCount) &&
+        showRemainingAvatars && (
+          <li className="flex items-center justify-center -ml-3 z-10">
+            <div className="relative">
+              <UserAvatar
+                user={{
+                  walletAddress: '0x0',
+                }}
+                size={size}
+                className={clsx(
+                  'border-base-white bg-base-white border rounded-full',
+                  {
+                    'border-2': size === 'xms',
+                  },
+                )}
+              />
+              <div
+                className={clsx(
+                  'placeholder absolute inset-0 rounded-full z-[1] bg-gray-50 flex items-center justify-center text-5 text-gray-700 border border-base-white text-center',
+                  {
+                    'border-2': size === 'xms',
+                  },
+                )}
+              >
+                {`+ ${remainingAvatarsCountProp || remainingAvatarsCount}`}
```

Comment:
What's the difference between the prop and the const in this component? How come we need 2 vars for the count?

Reply:
The first one - `remainingAvatarsCountProp` - is an optional prop, which you can pass to overwrite the value that is calculated based on avatars to show and items length - `remainingAvatarsCount`
Previous implementation required you to pass the whole members array. This additional prop lets you pass a smaller array with the number of members that should be displayed if there are more members.